packages/spacecat-shared-rum-api-client/src/common/traffic.js

```diff
@@ -160,6 +168,43 @@ const RULES = (domain) => ([
   { type: 'owned', category: 'uncategorized', referrer: any, utmSource: any, utmMedium: any, tracking: any },
 ]);
 
+export function extractTrafficHints(bundle) {
+  const findEvent = (checkpoint, source = '') => bundle.events.find((e) => e.checkpoint === checkpoint && (!source || e.source === source)) || {};
+
+  const referrer = findEvent('enter').source || '';
+  const utmSource = findEvent('utm', 'utm_source').target || '';
+  const utmMedium = findEvent('utm', 'utm_medium').target || '';
+  const tracking = findEvent('paid').checkpoint || findEvent('email').checkpoint || '';
+
+  return {
+    url: bundle.url,
+    weight: bundle.weight,
+    referrer,
+    utmSource,
+    utmMedium,
+    tracking,
+  };
+}
+
+/**
+ * Returns the name of the referrer as single word.
+ * For example: facebook instead of www.facebook.com
+ * @param {*} referrerString
+ */
+export function classifyReferrer(referrerString) {
```

Comment:
to classify the vendors, we need to maintain a vendor dictionary, let's not use what we use for type and category. To align with other RUM related products, we can use the same dictionary here: https://github.com/adobe/helix-website/blob/main/tools/oversight/acquisition.js#L12

Reply:
great, thank you @ekremney .
Is it ok to copy the `vendorClassifications` to our repo?

Reply:
@rpapani yes, definitely.